exercises/java/src/main/java/com/adaptionsoft/games/Board.java

```diff
@@ -0,0 +1,31 @@
+package com.adaptionsoft.games;
+
+import com.adaptionsoft.games.categories.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Board {
+
+    private Map<Integer, Category> positions = new HashMap<>();
+
+    public Map<Integer, Category> createBoard() {
```

Comment:
Interesting that this is returning a map rather than a reference to the Board object.  Feels like it might be a constructor masquerading as something else.